### ISSUE TITLE
KQueue reactor misspelled in reactorbuilder

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4556]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4556|
|Type|defect|
|Created|2010-07-07 20:26:24Z|
|Branch|https://github.com/twisted/twisted/tree/kqreactor-tests-4556|

```python
    _reactors = ["twisted.internet.selectreactor.SelectReactor",
                 "twisted.internet.pollreactor.PollReactor",
                 "twisted.internet.epollreactor.EPollReactor",
                 "twisted.internet.glib2reactor.Glib2Reactor",
                 "twisted.internet.gtk2reactor.Gtk2Reactor",
                 "twisted.internet.kqueuereactor.KQueueReactor",
                 "twisted.internet.win32eventreactor.Win32Reactor",
                 "twisted.internet.iocpreactor.reactor.IOCPReactor"]
```

```
exarkun@boson:~/Projects/Twisted/branches/bad-fileno-call-4539$ ls -l twisted/internet/kqreactor.py 
-rw-r--r-- 1 exarkun exarkun 7334 2010-05-23 12:22 twisted/internet/kqreactor.py
exarkun@boson:~/Projects/Twisted/branches/bad-fileno-call-4539$ ls -l twisted/internet/kqueuereactor.py
ls: cannot access twisted/internet/kqueuereactor.py: No such file or directory
exarkun@boson:~/Projects/Twisted/branches/bad-fileno-call-4539$ 
```

<details><summary>Searchable metadata</summary>

```
trac-id__4556 4556
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_kqreactor_tests_4556 branches/kqreactor-tests-4556
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1278534384000000 1278534384000000
changetime__1278899342000000 1278899342000000
version__None None
owner__ 
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [29664]) Branching to 'kqreactor-tests-4556'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Not enough to get KQueue reactor to run on any build slave, as far as I know.  But now at least it's a slave-side configuration issue (or a KQueue reactor sucks and is broken issue), not a the-test-harness-is-crap issue.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_3"></a>|@jesstess set owner to @exarkun|
|-|-|

Thanks for catching this, exarkun. Looks ready to merge modulo a NEWS file.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [29769]) Merge kqreactor-tests-4556

Author: exarkun
Reviewer: jesstess
Fixes: [#4556](https://github.com/twisted/twisted/issues/4556)

Correct the spelling of the KQueue reactor module in the generative reactor
tests fixture.  This makes it possible for certain tests to be run against
KQueue reactor (but no guarantees about whether anything will work).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
